src/goto-harness/memory_snapshot_harness_generator.h

```diff
@@ -117,6 +117,10 @@ class memory_snapshot_harness_generatort : public goto_harness_generatort
   /// \param goto_model: Model where the modification takes place
   void add_init_section(goto_modelt &goto_model) const;
 
+  const symbolt &fresh_symbol_copy(
```

Comment:
Please add documentation.

Reply:
@tautschnig ok, but the above comments should probably be reflected on the prerequisite PR #4261.